digdag-tests/src/test/java/acceptance/td/PyIT.java

```diff
@@ -149,7 +149,7 @@ public void testRunOnEcs()
             }
             assertThat(attempt.getSuccess(), is(true));
         }
-
+        Thread.sleep(60 * 1000); // Log will be delayed
```

Comment:
I think I saw the impl of EcsCommandExecutor waits for the end marker from CWL. So, missing log means a bug of `EcsCommandExecutor`?

Reply:
I am not sure the root cause, but the failure frequently happened and difficult to pass CI.
After adding the step, I don't face the issue. So log will be delayed.